ptf_tests/common/config/l3_action_selector_with_vhost_test1.json

```diff
@@ -82,7 +82,9 @@
             "name": "ingress.as_sl3",
      "member_details": [
       "action=ingress.send(0),member_id=1",
-      "action=ingress.send(1),member_id=2"
+      "action=ingress.send(1),member_id=2",
+      "action=ingress.send(2),member_id=3",

```

Comment:
RQM test procedure shows 2 vhost ports, 2 VMs and 2 members.  Please match the procedure.

Reply:
done